emukit/core/optimization/optimizer.py

```diff
@@ -0,0 +1,174 @@
+# Copyright (c) 2016, the GPyOpt Authors
```

Comment:
As the current maintainer of GPyOpt, I grant emukit permission to claim copyright to this file. Feel free to change to header to emukit's

side note: we may actually want to refactor this one quite a bit

Reply:
Refactor in this PR or future?

Reply:
definitely not in this pr, sorry for confusion